data/recommendation/notify.js

```diff
@@ -26,10 +23,23 @@ class Notify {
     });
   }
 
+  prettifyCount(ISODate) {
+    return ISODate.toString().replace(/\B(?=(\d{3})+(?!\d))/g, ',');
+  }
+
   render(options) {
     const div = yo`<div id="panel"></div>`;
     if (options.boxType === 'warn') {
-      div.appendChild(this.createWarningBox(options.site, options.year));
+      const date = new Date(this.recs.BreachDate);
+      const year = date.getFullYear();
+      div.appendChild(this.createWarningBox(this.recs.Title, year));
+    } else if (options.boxType === 'moreInfo') {
+      const date = new Date(this.recs.BreachDate);
+      const year = date.getFullYear();
+      const addedDate = new Date(this.recs.AddedDate);
+      const addedYear = addedDate.getFullYear();
+      const count = this.prettifyCount(this.recs.PwnCount);
+      div.appendChild(this.createInfoBox(this.recs.Title, year, count, addedYear));
```

Comment:
Formatting counts, parsing and splitting dates, and other UI concerns should probably be done in `createInfoBox` instead of here.